src/urh/dev/PCAPNG.py

```diff
@@ -0,0 +1,114 @@
+import os
+import struct
+import time
+import math
+
+from urh.util.Logger import logger
+
+from urh.signalprocessing.Message import Message
+
+
+class PCAPNG(object):
+    def __init__(self):
+        self.__messages = []
+        self.__messages_timestamps = []
+        self.__filename = ""
+
+    def __build_pcapng_shb(self, shb_userappl : str = "", shb_hardware : str = "") -> bytes:
+        BLOCKTYPE = 0x0A0D0D0A
```

Comment:
For reference, could you add a link to the doc as comment where these magic values are taken from?